src/components/ProxiesRenderWarpper.tsx

```diff
@@ -0,0 +1,45 @@
+import { ResolvedJSXElement, type ParentComponent } from 'solid-js'
+import { twMerge } from 'tailwind-merge'
+import { renderProxiesInTwoColumns } from '~/signals'
+
+export const ProxiesRenderWarpper: ParentComponent = (props) => {
+  const [width, setWidth] = createSignal(window.innerWidth)
+  const isBiggerScreen = createMemo(() => width() > 480)
+
+  const handleResize = () => {
+    setWidth(window.innerWidth)
+  }
+
+  onMount(() => {
+    window.addEventListener('resize', handleResize)
+    onCleanup(() => {
+      window.removeEventListener('resize', handleResize)
+    })
+  })
+
+  return () => {
+    const content = children(() => props.children) as () => ResolvedJSXElement[]
+    const filterContent = (target: number) => {
+      return content()
+        ?.filter((_: ResolvedJSXElement, index: number) => index % 2 === target)
+        .map((proxy) => {
+          return <div class="mb-2">{proxy}</div>
```

Comment:
It's safer to use `gap` here instead of `mb-x`

Reply:
The parent element is display: block and not work with gap-x